docs/configure/triggers.md

```diff
@@ -134,6 +152,39 @@ To configure your trigger by using **JSON** mode instead of **Builder** mode, pa
   ]
 ```
 
+{{% /tab %}}
+{{% tab name="JSON Template: Conditional Data Ingestion" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"triggers": [
+    {
+      "name": "<trigger name>",
+      "event": {
+        "type": "conditional_data_ingested",
+        "conditional": {
+          "data_capture_method": "<component>:<name-of-component>:<method>",
+          "condition": {
+            "evals": [
+              {
+                "operator": "<operator>",
+                "value": <object, string, or int>
```

Comment:
I don't actually see an example where this is string or int. Do you have an example for that? All the ones I tested just now use an object.

Reply:
I'm grabbing this from the scope doc:
<img width="703" alt="image" src="https://github.com/user-attachments/assets/f7476116-3a4b-48d5-832f-3ee8995c8fcc">
if you think it should just be object fine to change it but I would assume there are cases where this could be a string or int, like lt 20. I guess all of the examples we have use keys and I'm not sure how to construct one without them either

Reply:
If you, based on the docs you've written, cannot construct a conditional trigger with this info, then this info will likely not help our readers either. So I think that's a good indication that this isn't good enough documentation.

It could be that this changes from when the scope document was written. If you don't know of an example, and we have now identified this potential issue, what have you done to investigate this? And if you haven't investigated further than that, please have a think what you could do to try to investigate this on your own. Once you've done that, if you cannot figure it out, I am happy to help, but in the first instance it's the authoring writer's job to understand and double check what they're writing about.

Reply:
I just looked into all the triggers possible on my kitchen sink machine and I can't find anything that doesn't operate like a key value pair (+ besides option for no conditions). I also asked Roxy and she said that everything has keys for now, so I am going to proceed as such and edit these parts to treat the value as an object or nested object that provides a key-value pair for conditions.

Reply:
I think the no nesting example is actually an example for this but it's unclear what method this would be used with. i thought maybe voltage for power sensor would make sense but even there you have to specify a key ("volts") and it seems to possibly be usable for power sensors, for voltage for example.

Reply:
oh thank you! I tried a similar thing and it didn't send me any notifications so I'm glad you found this.

Reply:
I also think conditional data triggers do not work on Try Viam. Can you ask Roxy about this and otherwise ask in the octagon slack?

Reply:
Asked her over dm, will let you know